### ISSUE TITLE
Validation set

### ISSUE THREAD

**Issue by diff7** [yangdongchao/AcademiCodec#13]
Hello, 
I could not figure out which validation set was used for results in the paper?

Could you help?

**Comment by diff7** [comment]
Thank you for your reply. 

Do I understand correctly that validation on as LibriTTS, VCTK and AISHELL would inappropriate because entire datasets  were used during training?
